module/libs/checks.py

```diff
@@ -135,7 +137,7 @@ def check_snmp(check, arguments, db_client, task_queue, result_queue):
             mapping_task['host'] = snmp_info.address
             # Get concurrency
             mapping_task['no_concurrency'] = serv.get('no_concurrency', False)
-            mapping_task['data'] = {"authData": cmdgen.CommunityData(snmp_info.community),
+            mapping_task['data'] = {"authData": cmdgen.CommunityData(communityIndex=snmp_info.community, communityName=snmp_info.community, mpModel=int(snmp_info.version)-1),
```

Comment:
Can you split the your lines at the communityName parameter to make them more readable, please?